### PR TITLE
fix token decimal bug

### PR DESCRIPTION
**Pull request by Preston-Harrison**

### CODE REVIEW

**Comment by Preston-Harrison** [comment]
<!channel> please review

**Review by bitcoinbrisbane — APPROVED** [review]
LGTM.  Seems odd the unit tests didnt find this bug to start with.

File: src/components/Convert/routes/psm.ts [hunk 1]
```diff
@@ -69,15 +69,16 @@ export const psmQuoteHandler = async (input: ConvertQuoteInput): Promise<Quote>
     HlpConfig.BASIS_POINTS_DIVISOR
   ).div(TRANSACTION_FEE_UNIT);
 
+  // converts 1 of the from token to 1 of the to token, accounting for decimals
```
Comment (rellfy) [inline src/components/Convert/routes/psm.ts:72]: Please make an util function to change the number of decimals.
```
const transformDecimals = (value: BigNumber, fromDecimals: number, toDecimals: number): BigNumber
```
and then you can unit test this function. I think we had this in the previous SDK but not this one.

**Comment by Preston-Harrison** [comment]
> LGTM. Seems odd the unit tests didnt find this bug to start with.

tests were written incorrectly

**Review by rellfy — CHANGES_REQUESTED** [review]

**Comment by Preston-Harrison** [comment]
Addressed the changes

**Review by nabsta-eth — APPROVED** [review]

**Pull request merged**